plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergParquetFileWriter.java

```diff
@@ -77,4 +79,40 @@ public Metrics getMetrics()
     {
         return fileMetrics(inputFile, metricsConfig);
     }
+
+    @Override
+    public long getWrittenBytes()
+    {
+        return parquetFileWriter.getWrittenBytes();
+    }
+
+    @Override
+    public long getMemoryUsage()
+    {
+        return parquetFileWriter.getMemoryUsage();
+    }
+
+    @Override
+    public void appendRows(Page dataPage)
+    {
+        parquetFileWriter.appendRows(dataPage);
+    }
+
+    @Override
+    public Closeable commit()
+    {
+        return parquetFileWriter.commit();
+    }
+
+    @Override
+    public void rollback()
+    {
+        parquetFileWriter.rollback();
+    }
+
+    @Override
+    public long getValidationCpuNanos()
+    {
+        return parquetFileWriter.getValidationCpuNanos();
```

Comment:
should `getVerificationTask` be forwarded as well?

Reply:
`getVerificationTask` is unused so I just removed it in a prep commit